test/integration/controller/jobset_controller_test.go

```diff
@@ -745,14 +1305,29 @@ func completeJob(job *batchv1.Job) {
 			Status: corev1.ConditionTrue,
 		}),
 	})
+	gomega.Eventually(func() error {
+		var jobGet batchv1.Job
+		if err := k8sClient.Get(ctx, types.NamespacedName{Name: job.Name, Namespace: job.Namespace}, &jobGet); err != nil {
+			return err
+		}
+		jobGet.Status.Succeeded = ptr.Deref(jobGet.Spec.Parallelism, 0)
+		return k8sClient.Status().Update(ctx, &jobGet)
+	}, timeout, interval).Should(gomega.Succeed())
+
 }
 
-func ReadyJob(job *batchv1.Job) {
+func readyJob(job *batchv1.Job) {
 	updateJobStatus(job, batchv1.JobStatus{
 		Ready: job.Spec.Parallelism,
 	})
 }
 
+func activeJob(job *batchv1.Job) {
```

Comment:
Can we create another function called `activeReplicatedJob` that calls `activeJob`.  `activeReplicatedJob` takes two parameters a list of jobs and a string representing the replicatedJob name that we want to activate. The function iterates over the list of jobs and only activates the ones with a matching replicatedJob name.

Reply:
updated.